app/store/models/Membership.d.ts

```diff
@@ -11,3 +12,9 @@ export default interface Membership {
   emailListsEnabled: boolean;
   createdAt: Dateish;
 }
+
+export interface PastMembership extends Membership {
+  startDate: Dateish;
+  endDate: Dateish;
+  abakusGroup: PublicGroup;
```

Comment:
Don't we have a specific group type for groups fields in models or something?

Reply:
Yup, but it's not used here. This one uses `PublicAbakusGroupSerializer`🙃